easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
```

Comment:
I used convention from #1973. You had no objections against naming convention and I thought that it will be new toolchain name.